src/fullcalendar/eventSourceFunction.js

```diff
@@ -52,6 +52,10 @@ export function eventSourceFunction(calendarObjects, calendar, start, end, timez
 				classNames.push('fc-event-nc-tentative')
 			}
 
+			if (object.getAlarmList().length > 0) {
```

Comment:
```suggestion
			if (object.hasComponent('VALARM')) {
```

Does the same without actually loading all the alarms :)